plugins/scaffolder-actions/scaffolder-backend-module-http-request/src/actions/run/helpers.ts

```diff
@@ -45,23 +44,22 @@ export const http = async (
   continueOnBadResponse: boolean = false,
 ): Promise<any> => {
   let res: any;
+  const TIMEOUT = options.timeout || 60_000;
   const controller = new AbortController();
-  const timeoutId = setTimeout(() => controller.abort(), DEFAULT_TIMEOUT);
+  const timeoutId = setTimeout(() => controller.abort(), TIMEOUT);
   const { url, ...other } = options;
   const httpOptions = { ...other, signal: controller.signal };
 
   try {
     res = await fetch(url, httpOptions);
-    if (!res) {
-      throw new HttpError(
-        `Request was aborted as it took longer than ${
-          DEFAULT_TIMEOUT / 1000
-        } seconds`,
-      );
-    }
   } catch (e) {
     throw new HttpError(`There was an issue with the request: ${e}`);
   }
+  if (!res) {
```

Comment:
This can only occur in a testing environment. The spec says that `fetch` always returns a `promise`.